app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -426,15 +426,16 @@ BrowserWorld::State::UpdateControllers(bool& aRelayoutWidgets) {
 
 
     const bool pressed = controller.buttonState & ControllerDelegate::BUTTON_TRIGGER ||
-                         controller.buttonState & ControllerDelegate::BUTTON_TOUCHPAD;
+                         controller.buttonState & ControllerDelegate::BUTTON_TOUCHPAD ||
+                         controller.buttonState & ControllerDelegate::BUTTON_A ||
+                         controller.buttonState & ControllerDelegate::BUTTON_X;
     const bool wasPressed = controller.lastButtonState & ControllerDelegate::BUTTON_TRIGGER ||
-                            controller.lastButtonState & ControllerDelegate::BUTTON_TOUCHPAD;
+                            controller.lastButtonState & ControllerDelegate::BUTTON_TOUCHPAD ||
+                            controller.buttonState & ControllerDelegate::BUTTON_A ||
+                            controller.buttonState & ControllerDelegate::BUTTON_X;;
```

Comment:
Ups I found a bug here, just before landing. You should use `lastButtonState` for A & X